auth-web/src/assets/scss/base.scss

```diff
@@ -184,3 +184,8 @@ legend {
 .gray-9 {
   color: $gray9;
 }
+
+.background-green {
+  background: $app-background-green;
+  border-color: $app-background-green;
+}
```

Comment:
I declared this here so BaseVDataTable can see it (even though we reference it in AffiliatedEntityTable).